lavalink/client.py

```diff
@@ -211,14 +211,15 @@ def add_node(self, host: str, port: int, password: str, region: str,
         else:
             return
 
-    def on(self, event: Event = "Generic"):
+    def on(self, event: Event = 'Generic'):
```

Comment:
Not sure how I feel about *wanting* an `Event` but receiving `str` instead (mixed types). Perhaps this can be `None` (to indicate "No specific event) which defaults to Generic within the code?